ui/dashboards/src/components/DashboardToolbar/DashboardToolbar.tsx

```diff
@@ -49,30 +49,34 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
   return (
     <>
       {isEditMode ? (
-        <Stack spacing={2}>
-          <Box sx={{ backgroundColor: (theme) => theme.palette.primary.light + '20' }}>
-            <Box padding={2} display="flex">
-              {dashboardTitle}
-              <Stack direction="row" spacing={1} marginLeft="auto">
-                <Button variant="contained" onClick={onSave}>
-                  Save
-                </Button>
-                <Button variant="outlined" onClick={onCancelButtonClick}>
-                  Cancel
-                </Button>
-              </Stack>
-            </Box>
+        <Stack spacing={1}>
+          <Box p={2} display="flex">
+            {dashboardTitle}
+            <Stack direction="row" spacing={1} marginLeft="auto">
+              <Button variant="contained" onClick={onSave}>
+                Save
+              </Button>
+              <Button variant="outlined" onClick={onCancelButtonClick}>
+                Cancel
+              </Button>
+            </Stack>
           </Box>
           <Box
             sx={{
               display: 'flex',
               width: '100%',
               alignItems: 'flex-start',
-              padding: (theme) => theme.spacing(2),
+              padding: (theme) => theme.spacing(0, 2, 2, 2),
             }}
           >
             <ErrorBoundary FallbackComponent={ErrorAlert}>
-              <TemplateVariableList initialVariableIsSticky={initialVariableIsSticky} />
+              <TemplateVariableList
+                initialVariableIsSticky={initialVariableIsSticky}
+                sx={{
+                  backgroundColor: ({ palette }) =>
+                    palette.mode === 'dark' ? palette.background.default : palette.background.paper,
```

Comment:
This seems a little weird that we're using different theme tokens here for the background in light vs dark. Maybe something to follow-up with @foxbat07 on?